augur/tasks/git/facade_tasks.py

```diff
@@ -282,13 +275,29 @@ def analyze_commits_in_parallel(repo_git, multithreaded: bool)-> None:
 
 
             #logger.info(f"Got to analysis!")
-            absoulte_path = get_absolute_repo_path(session.repo_base_directory, repo.repo_id, repo.repo_path,repo.repo_name)
-            repo_loc = (f"{absoulte_path}/.git")
+            commitRecords = analyze_commit(session, repo_id, repo_loc, commitTuple)
+            #logger.debug(commitRecord)
+            if len(commitRecords):
+                pendingCommitRecordsToInsert.extend(commitRecords)
+                if len(pendingCommitRecordsToInsert) >= 1000:
+                    facade_bulk_insert_commits(session,pendingCommitRecordsToInsert)
+                    pendingCommitRecordsToInsert = []
+
         
+        facade_bulk_insert_commits(session,pendingCommitRecordsToInsert)
+
+        
+
 
-            analyze_commit(session, repo_id, repo_loc, commitTuple)
+    # Remove the working commit.
+    remove_commit = s.sql.text("""DELETE FROM working_commits 
+    	WHERE repos_id = :repo_id AND working_commit IN :hashes
+    	""").bindparams(repo_id=repo_id,hashes=tuple(queue))
+    session.execute_sql(remove_commit)  
```

Comment:
This query to remove the working commits appears to be at the wrong indentation level, which might be causing silent issues

Reply:
Yeah this should be indented will fix